src/utils/internalLinks.js

```diff
@@ -1,5 +1,6 @@
 /* @flow strict-local */
 import { addBreadcrumb } from '@sentry/react-native';
+import { by_stream_topic_url } from '@zulip/shared/js/internal_url';
```

Comment:
Let's do this import in the same style as it's done in the web app, because the names are often chosen with that in mind. So:
```suggestion
import * as internal_url from '@zulip/shared/js/internal_url';
```

It looks like we do that in one of the existing places we use shared code (importing `typing_status`), but not another (importing `poll_data`). I should probably go fix it for the latter and add to the style guide.

Reply:
https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#import-shared
https://github.com/zulip/zulip-mobile/pull/5276